src/transport_ofi.h

```diff
@@ -214,10 +214,10 @@ int shmem_transport_fini(void);
 
 extern size_t SHMEM_Dtsize[FI_DATATYPE_LAST];
 
-static inline void shmem_transport_get_wait(void);
+static inline void shmem_transport_get_wait(shmem_ctx_t ctx);
 
 static inline
-void shmem_transport_ofi_drain_cq(void)
+void shmem_transport_ofi_drain_cq()
```

Comment:
Removing void here will cause GCC to produce a warning about the function not being properly prototyped.

Reply:
Whoops!